extra-multimedia/kodi-pvr-iptvsimple/autobuild/defines

```diff
@@ -0,0 +1,5 @@
+PKGNAME="kodi-pvr-iptvsimple"
+PKGDES="Basic PVR Addon for Kodi to receive IPTV Live TV, Radio channels, and EPG directly in Kodi"
```

Comment:
“ Basic PVR Addon for Kodi to receive IPTV Live TV, Radio channels, and EPG” will suffice.